PaneViewController/PaneViewController.swift

```diff
@@ -352,9 +348,6 @@ open class PaneViewController: UIViewController {
     open override func viewWillTransition(to size: CGSize, with coordinator: UIViewControllerTransitionCoordinator) {
         super.viewWillTransition(to: size, with: coordinator)
 
-        guard UIApplication.shared.applicationState == .active
```

Comment:
This fix was put in to fix a different issue that was happening, I think when the app is being backgrounded. I can't remember what the bug was... but there was a reason for it. Maybe its the wrong fix, and your change makes it work in all cases, but I'm hoping @samuelbradshaw might remember a bug with sidebar when closing the app, maybe on iPad?

Reply:
Yeah, I though there must be a reason, because it's not something you'd normally check here.

Reply:
https://jira.churchofjesuschrist.org/browse/GLI-6006

Reply:
I'm not seeing that issue 👆in my testing. I'll test it on iPad to be sure...

Reply:
That looks like it. I think what was happening is we're responding to those trait collection changed callbacks and that triggers it to dismiss the sidebar if I remember right

Reply:
Yes, there have been a few issues in the past with the sidebar not staying open when you switch apps, switch screens, etc.

Reply:
Does rotating close the sidebar? Ideally it shouldn't, at least on an iPad.

Reply:
It does. That could be part of the problem. I'll look at it.

Reply:
During the rotation handling there's a part that closes then re-opens the sidebar immediately (I'm not sure why, but I assume it's needed). The problem is, within the close method it signals that it closed, and the app in turn pauses video and then makes sure it's closed. I'm changing it so that it doesn't signal in that case (where it's closing and immediately re-opening). That fixes the problem with the sidebar closing on rotation and when the app is backgrounded.